polyfill/test/duration.mjs

```diff
@@ -1419,7 +1423,7 @@ describe('Duration', () => {
           const partialMonth = (daysPastJuly1 + partialDay) / 31;
           const totalMonths = 5 * 12 + 5 + 1 + partialMonth; // +1 for 5 weeks
           const total = d.total({ unit: 'months', relativeTo });
-          assert(Math.abs(total - totalMonths) < Number.EPSILON); // 66.32930780242619
+          equal(total.toPrecision(15), totalMonths.toPrecision(15)); // 66.32930780242619
```

Comment:
Are these tests changed because they no longer passed? Or just because the assertion message is better when using `.toPrecision()`?

Reply:
If I remember correctly, it was the former. A few tests failed if they were using comparison values that had accumulated floating-point errors, so instead of changing the test data to make it exact I just limited the precision of the comparison, per the comment down at line 1530 which applies here too. LMK if you want to see changes.